src/port.js

```diff
@@ -31,6 +34,28 @@ export default class Port {
 			console.log(msg);
 		}
 	}
+	getService(serviceType, version) {
+		if(!this.isConnected) {
+			throw new Error('Cannot getService() before connect() has completed');
+		}
+		let me = this;
+		return new Promise((resolve, reject) => {
+			me.requestRaw(`service:${serviceType}:${version}`)
```

Comment:
Haha easier said than done since all the variables (serviceType, version, resolve, methodNames) need to be in scope most of the way down the chain.